Src/CSharpier/DocPrinter/DocPrinter.cs

```diff
@@ -194,7 +193,7 @@ void Push(Doc doc, PrintMode printMode, Indent indent)
                         {
                             // TODO 1 new line stuff
                             TrimOutput(output);
-                            output.Append(newLine + command.Indent.Value);
+                            output.Append(endOfLine + command.Indent.Value);
```

Comment:
Nit: it's faster to call `StringBuilder.Append()` twice than doing a string concat in its argument